README.md

```diff
@@ -180,8 +180,10 @@ npm install jsondiffpatch
 ```
 
 ```js
-var jsondiffpatch = require('jsondiffpatch').create(options);
+var jsondiffpatch = require('jsondiffpatch');
+var jsondiffpatchInstance = jsondiffpatch.create(options);
 ```
+Some properties are available only from static main module (e.g. formatters, console), so we need to keep the reference to it if we want to use them.
```

Comment:
The formatter wants a new line here:

```suggestion
var jsondiffpatch = require('jsondiffpatch');
var jsondiffpatchInstance = jsondiffpatch.create(options);
```

Some properties are available only from static main module (e.g. formatters, console), so we need to keep the reference to it if we want to use them.
```